tests/infra_test.go

```diff
@@ -696,6 +697,9 @@ var _ = Describe("[Serial]Infrastructure", func() {
 			for ix := 0; ix < concurrency; ix++ {
 				err := <-errors
 				if err != nil {
+					urlErr, ok := err.(*netURL.Error)
+					Expect(ok).To(BeTrue())
+					Expect(urlErr.Err.Error()).To(ContainSubstring("Client.Timeout exceeded while awaiting headers"))
```

Comment:
ditto

Reply:
in case of error the string would be redundant because the error will say it already

Reply:
fair enough